consensus/common_test.go

```diff
@@ -278,7 +298,8 @@ func newConsensusStateWithConfigAndBlockStore(thisConfig *cfg.Config, state sm.S
 	evpool := sm.MockEvidencePool{}
 
 	// Make ConsensusState
-	stateDB := dbm.NewMemDB()
+	stateDB := blockDB
```

Comment:
this is super confusing. please add a comment

Reply:
can't we add yet another argument to `newConsensusStateWithConfigAndBlockStore` function - `stateDB dbm.DB`